OctoWS2811.h

```diff
@@ -27,51 +27,81 @@
 #include <Arduino.h>
 #include "DMAChannel.h"
 
-#if TEENSYDUINO < 120
-#error "Teensyduino version 1.20 or later is required to compile this library."
+#if TEENSYDUINO < 121
+#error "Teensyduino version 1.21 or later is required to compile this library."
 #endif
 #ifdef __AVR__
-#error "The Audio Library only works with Teensy 3.X.  Teensy 2.0 is unsupported."
+#error "OctoWS2811 does not work with Teensy 2.0 or Teensy++ 2.0."
```

Comment:
My mistake.  I started with an older version and only recently decided to share my changes in GitHub.  I also recently upgraded to a newer version I got off PJRC which had the above #if TEENSYDUINO < 121 change.  So combining the PJRC version with my older change then merging it with the GitHub version is how I ended up with the version I have in GitHub.